packages/text-field/src/vaadin-input-field-shared-styles.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
Is the plan to move the other field types (number, email, area) into this package as well, and reuse this style module?

If not I would suggest to rename to module to something more specific to `vaadin-text-field`.

Reply:
Yes, the plan is that all the field components e.g. `vaadin-combo-box` and `vaadin-date-picker` will use it.